sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs

```diff
@@ -3039,6 +3198,27 @@ public partial class RestorableSqlResourceData : Azure.ResourceManager.Models.Re
         public System.Collections.Generic.IReadOnlyList<string> CollectionNames { get { throw null; } }
         public string DatabaseName { get { throw null; } }
     }
+    public partial class RestorableTableGetResult : Azure.ResourceManager.Models.ResourceData
+    {
+        internal RestorableTableGetResult() { }
+        public Azure.ResourceManager.CosmosDB.Models.RestorableTablePropertiesResource Resource { get { throw null; } }
+    }
+    public partial class RestorableTablePropertiesResource
+    {
+        internal RestorableTablePropertiesResource() { }
+        public string EventTimestamp { get { throw null; } }
+        public Azure.ResourceManager.CosmosDB.Models.CosmosDBOperationType? OperationType { get { throw null; } }
+        public string OwnerId { get { throw null; } }
+        public string OwnerResourceId { get { throw null; } }
+        public string Rid { get { throw null; } }
+    }
+    public partial class RestorableTableResourcesGetResult
+    {
+        internal RestorableTableResourcesGetResult() { }
+        public string Id { get { throw null; } }
+        public string Name { get { throw null; } }
+        public string ResourceType { get { throw null; } }
```

Comment:
Is the format of this property a "ARM resource type" (something like `Microsoft.DocumentDB/something`)? If so, please specify the corresponding configuration to change the type of this property to `ResourceType`.
If it is not, please specify what kind of type this is, and we need to rename this property

Reply:
We already have this change in preview version and we are just making it stable. Also, we are following similar convention for existing APIs of sql and mongo so we decided to use same for these new ones as well to remain consistent.